docs/data/rpc/api-reference/methods/getEvents.mdx

```diff
@@ -4,5 +4,8 @@ description: Returns contract events
 ---
 
 import { RpcMethod } from "@site/src/components/RpcMethod";
+import rpcSpec from "@site/static/stellar-rpc.openrpc.json";
 
-<RpcMethod method="getEvents" platform="soroban" />
+<RpcMethod
+  method={rpcSpec.methods.filter((meth) => meth.name === "getEvents")[0]}
```

Comment:
Could it be refactored so it's possible to call in like before?
`<RpcMethod method="getEvents">`

Reply:
The short answer is "I'm not sure if we can."

The reason for co-locating the specfile _into_ the `/platforms/anchor-platform` directory is to make it a part of the versioned docs so when someone clicks from v2 to v3, the proper specfile gets rendered and displayed.

But, the `RpcMethod` React component lives entirely outside of that context. I'm not sure how to make the component work across differing versions (or non-versioned like stellar-rpc) while still passing in `getEvent`-like strings like we had before (which i do agree has much better ergonomics). I'm open to ideas or suggestions, for sure, I was just drawing a blank.

Reply:
> to make it a part of the versioned docs so when someone clicks from v2 to v3, the proper specfile gets rendered and displayed.

Got it, I didn't think about it, makes sense! 
Not sure if it can be done in a better manner then 😔